src/test/scala/org/wikimedia/analytics/hdfstools/TestHdfsRsyncHelper.scala

```diff
@@ -7,11 +7,10 @@ import org.apache.commons.io.FileUtils
 import org.apache.log4j.spi.LoggingEvent
 import org.apache.log4j.{AppenderSkeleton}
 import org.scalatest.BeforeAndAfterEach
-import org.scalatest.flatspec.AnyFlatSpec
-import org.scalatest.matchers.should.Matchers
+import org.scalatest.{FlatSpec, Matchers}
 
 
-trait TestHdfsRsyncHelper extends AnyFlatSpec with Matchers with BeforeAndAfterEach {
+trait TestHdfsRsyncHelper extends FlatSpec with Matchers with BeforeAndAfterEach {
```

Comment:
If we don't change the version of scalatest we can keep this to `AnyFlatSpec

Reply:
Thanks for this. New commit will have this change